v2/operations/receipt_test.go

```diff
@@ -0,0 +1,29 @@
+package operations_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/omise/omise-go/v2"
+	"github.com/omise/omise-go/v2/internal/testutil"
+	a "github.com/stretchr/testify/assert"
+)
+
+func TestReceipt(t *testing.T) {
+	const (
+		ReceiptID = "rcpt_test_12345"
+	)
+
+	client := testutil.NewFixedClient(t)
+	receipt, _ := client.Receipt().Retrieve(context.Background(), &omise.RetrieveReceiptParams{ReceiptID})
+	a.Equal(t, ReceiptID, receipt.ID)
```

Comment:
should check first if `receipt` is not `nil` if using `assert` package, not `require` here
otherwise when this fails, everything else will be failed too and produce a lot of useless failure messages since the root cause is `nil`.